### DISCUSSION TITLE
External power supply to use with AUX connector

### DISCUSSION THREAD

**Discussion by nicolas-r** [tjko/fanpico#125] [Q&A]
Hello

I would like to use the fan pico without any connection to a system to get the power and I wonder what I need to power it up using the AUX connector ? I have some usb power supply that control some fans, is there something equivalent that can power the pico ?

Regards

Nicolas

**Comment by tjko** [comment]
To power FanPico via AUX connector you will need a 12V (DC) power supply (ideally one that can put out 3A or so for 0804D and a 2A one  for 0401D model).

AUX connector uses standard PC Floppy power connector (sometimes referred as "Berg" connector): https://en.wikipedia.org/wiki/Berg_connector

You only need to connect +12V (pin 4) and GND (pin 2 and/or 3).


Note, FanPico-0401D model supports installing either Floppy connector or standard 5.5mm DC connector as the AUX connector.

<img width="400" alt="fanpico-0401d-dc-connector" src="https://github.com/user-attachments/assets/1ea984c1-7c7d-434a-8e2b-2b3869056593">

**Comment by tjko** [comment]
Some external IDE/SATA hard drive to USB adapters come with external Power Supply that has either SATA or older 4-pin "Molex" connector.   Those work as well with standard PC adapter cables like these:

<img width="276" alt="molex to berg" src="https://github.com/user-attachments/assets/83f84db7-c20d-48a1-a603-2fdfbd0f1624">
<img width="280" alt="sata to berg" src="https://github.com/user-attachments/assets/d3f47bfb-0076-4da9-bf96-3e3ff5b80efc">

**Comment by nicolas-r** [comment]
Hi

Thanks for the answer, I'll see what I can find

Nicolas

**Comment by tjko** [comment]
You can also build adapter from DC connector to Floppy/Berg connector to power FanPico:

<img width="400" alt="DC to Berg connector adapter" src="https://github.com/user-attachments/assets/82ab1e39-2bd9-423b-857d-db0eddf20feb">

I've been making above style adapters when need to user regular 12V DC power supply.    Adapter shown above was made from a DC-connector pigtail cable by crimping Floppy connector terminals to it and then inserting pins to a connector housing.

One place to get the Floppy connector terminals and housing: https://www.aliexpress.us/item/3256801603097051.html  (I have no affiliation with above seller, I've just ordered some terminals and housings and they've seemed to be "ok" quality).
